lib/modules/subredditManager.js

```diff
@@ -438,6 +438,8 @@ function showSubredditGroupDropdown(obj) {
 }
 
 function _showSubredditGroupDropdown(obj, subreddits, suffix) {
+	// Updates source object referenced on the 'click' events
+	subredditGroupDropdownRefItem = obj;
```

Comment:
I think I had originally just added a variable to the `$subredditGroupDropdown` containing the object. Essentially: `$subredditGroupDropdown.selectedItem = obj;`, but it didn't match the existing workflow, so I went with that.